topics/ui/controls/tabs.md

```diff
@@ -24,49 +25,71 @@ Follow the rules in [Groups of controls](groups_of_controls.md).
 
 ## How to use
 
-Use title capitalization for tab labels.
+### Label
 
-Make the label short, preferably no more than 3 words.
+Use title capitalization for tab labels. Make the label short, preferably no more than 3 words.
 Avoid generic words, such as "General" or "Advanced".
 See [Writing short and clear](writing_short.md).
 
-![](02_naming.png){width=284}
+<table style="none" border="false">
+  <tr>
+<td width="50%">
+      <format color="Green" style="bold">Correct</format><img src="02_How_to_use_correct.png" alt="A correct example of a tab label"/>
+    </td>
+    <td width="50%">
+      <format color="Red" style="bold">Incorrect</format><img src="02_How_to_use_incorrect.png" alt="An incorrect example of a tab label"/>
+    </td>
+  </tr>
+</table>
+
+###  Organization and Accessibility
 
 Place the most frequently used content in the first tab.
+
 Tabs that do not fit allotted screen space automatically hide under the dropdown component.
 (It is better to add no more than 8 tabs, but this number is not limited.)
 
-![](03_hidden_tabs.png){width=533}
+![Hiding tabs that do not fit](03_How_to_use.png){width=706}
 
-Always place tabs on top. It is possible to place them at other sides of the content — bottom, left, or right — but such a placement is extremely rare and might confuse users.
+### Unavailable content
 
 Do not remove or disable a tab when its function is unavailable. Explain why a tab’s content is unavailable in the body of the tab.
 
 ## Placement
 
-Make sure the border of the tab reaches the edges of the area tabs occupy.
+### Position
 
-<format color="Red" style="bold">Incorrect</format>
+Always place tabs on top. It is possible to place them at other sides of the content — bottom, left, or right — but such a placement is extremely rare and might confuse users.
```

Comment:
When reading the first time, had the question "on top of what?". Possibly write as "Always place tabs on top of the content", and remove "of the content" in the 2nd sentence.